FLAIR.md

```diff
@@ -0,0 +1,29 @@
+Acknowledgments
+=========
+
+<p><a href="https://www.browserstack.com/"><img alt="" valign="middle" src="http://media.archiveofourown.org/ao3/logos/Browserstack.svg" width="200"/> BrowserStack</a> for cross-browser testing tool.</p>
```

Comment:
I feel like there's a word missing here or something. Maybe just "for cross-browser testing" or "for our cross-browser testing tool?"